frontend_tests/node_tests/compose.js

```diff
@@ -256,6 +257,76 @@ people.add(bob);
     assert($("#compose-all-everyone").visible());
 }());
 
+(function teste_markdown_shortcuts() {
```

Comment:
Can you change "teste_" to "test_" (typo)

Reply:
yeah, nice catch.